app/controllers/my_account_controller.rb

```diff
@@ -4,8 +4,14 @@ class MyAccountController < ApplicationController
   def destroy
     @user =
       User.includes(
-        logs: %i[log_entries log_shares],
+        :auth_tokens,
```

Comment:
Not all of these additions were required to get specs passing, and, even with all of these additions, I think that this will still fail for my user in development mode. I am going to try to iterate on this more in future PRs. Here's one relevant ticket: https://davidrunger.atlassian.net/browse/DEV-208 .